cms/templates/asset_index.html

```diff
@@ -35,32 +35,31 @@
 
 <div class="wrapper-mast wrapper">
     <header class="mast has-actions has-subtitle">
-        <h1 class="page-header">
+        <h3 class="page-header">
             <small class="subtitle">${_("Content")}</small>
-            <span class="sr">&gt; </span>${_("Files & Uploads")}
-        </h1>
+            <span class="sr">- </span>${_("Files & Uploads")}
```

Comment:
@cptvitamin explained that it's purpose it to tell a person using a screen reader that `Files & Uploads` is a subset of `Content`.